airflow/providers/amazon/aws/triggers/rds.py

```diff
@@ -18,11 +18,17 @@
 
 from typing import Any
 
+from deprecated import deprecated
+
+from airflow.providers.amazon.aws.hooks.base_aws import AwsGenericHook
 from airflow.providers.amazon.aws.hooks.rds import RdsHook
+from airflow.providers.amazon.aws.triggers.base import AwsBaseWaiterTrigger
+from airflow.providers.amazon.aws.utils.rds import RdsDbType
 from airflow.providers.amazon.aws.utils.waiter_with_logging import async_wait
 from airflow.triggers.base import BaseTrigger, TriggerEvent
 
 
+@deprecated(reason="Use the other specialized RDS triggers")
```

Comment:
Should we add a guide somewhere so that the users have a better idea of what `other specialized RDS triggers` mean and how to migrate?

Reply:
ah there was a comment recently by @eladkal  that we shouldn't use those deprecated annotations in providers, so I'll change it.
But my honest opinion (backed by absolutely no data) on this is that there is no user dealing with this trigger directly, so writing a detailed guide would be a waste of time (and also, they are here in the same file, it's not too complex to sort it out if you are the kind of user to write your own triggers by deriving this one)

Reply:
Got it. Sounds good 👍